docs/rfcs/0002/readme.md

```diff
@@ -37,21 +34,58 @@ The proposal below might take a couple of read-throughs; I've also added a concr
 2. The AUTHOR works with the ADMINISTRATOR to define:
     1. a unique name for the program to be referred by USER
     2. the pod template specification for executing their program
-    3. the NNF storage requirements described above. 
+    3. the NNF storage requirements described above.
 3. The ADMINISTRATOR creates a corresponding _NNF Container Profile_ custom kubernetes resource with the necessary NNF storage requirements and pod specification as described by the AUTHOR
-4. The USER who desires to use the application works with the AUTHOR and the related NNF Container Profile to understand the storage requirements.
+4. The USER who desires to use the application works with the AUTHOR and the related NNF Container Profile to understand the storage requirements
 5. The USER submits a WLM job with the #DW container fields populated
 6. WLM runs the job and drives the job through the following stages...
-    1. Proposal: RABBIT validates the #DW container directive by comparing the supplied values to what is listed in the NNF Container Profile. If the USER fails to meet the requirements, the job fails. 
-    2. Pre-run: RABBIT software will:
-        1. create a config map reflecting the storage requirements and any runtime parameters; this is provided to the container at the volume mount named "nnf-config", if specified.
+    1. `Proposal`: RABBIT validates the #DW container directive by comparing the supplied values to what is listed in the NNF Container Profile. If the USER fails to meet the requirements, the job fails
+    2. `PreRun`: RABBIT software will:
+        1. create a config map reflecting the storage requirements and any runtime parameters; this is provided to the container at the volume mount named `nnf-config`, if specified
         2. duplicate the pod template specification from the Container Profile and patches the necessary Volumes and the config map. The spec is used as the basis for starting the necessary pods and containers.
-    3. The containerized application executes. The expected mounts are available per the requirements and celebration occurs.
+    3. The containerized application(s) executes. The expected mounts are available per the requirements and celebration occurs. The pods will continue to run until:
+       1. the pod completes successfully (any failed pods will be restarted)
+       2. timeout (i.e. `activeDeadlineSeconds`) is hit (optional)
+       3. the max number of pod retries (i.e. `backoffLimit`) is hit (indicating failure on all retry attempts)
+          1. Note: retry limit is non-optional per Kubernetes configuration
+          2. If retries are not desired, this number could be set to disable any retry attempts.
```

Comment:
What is the value used to disable retry?

Reply:
Setting this to 0 will disable any retries. I'll add that.